crates/debugger_ui/src/debugger_panel.rs

```diff
@@ -272,6 +272,24 @@ impl DebugPanel {
         })
     }
 
+    fn handle_continued_event(
+        this: &mut Self,
+        client_id: &DebugAdapterClientId,
+        event: &ContinuedEvent,
+        cx: &mut ViewContext<Self>,
+    ) {
+        let all_threads = event.all_threads_continued.unwrap_or(false);
+        let client = this.debug_client_by_id(client_id.clone(), cx);
+
+        if all_threads {
+            for thread in client.thread_states().values_mut() {
+                thread.status = ThreadStatus::Running;
+            }
+        } else {
+            client.update_thread_state_status(event.thread_id, ThreadStatus::Running);
+        }
```

Comment:
I think we have to notify here at the end, to make sure all the tabs are re-rendered.

Reply:
Done, thanks!